exasol_integration_test_docker_environment/lib/test_environment/setup_external_database_host.py

```diff
@@ -60,8 +60,8 @@ def setup_database(self):
 
     def get_xml_rpc_object(self, object_name: str = ""):
         uri = 'https://{user}:{password}@{host}:{port}/{cluster_name}/{object_name}'.format(
-            user=quote_plus(self.external_exasol_xmlrpc_user),
-            password=quote_plus(self.external_exasol_xmlrpc_password),
+            user=quote_plus(self.external_exasol_xmlrpc_user or ""),
```

Comment:
Does it really make sense to assume empty string here, I would have said check and throw exception is more correct

Reply:
Done